go.mod

```diff
@@ -1,12 +1,14 @@
 module github.com/consensys/gnark
 
-go 1.20
+go 1.21
+
+toolchain go1.21.4
```

Comment:
heh didn't know this one. Why did you need to add that? :)

Reply:
I think `go get` adds it automatically.